translations/default/AdminCatalogHelp.xlf

```diff
@@ -358,8 +358,8 @@
   <file original="src/PrestaShopBundle/Form/Admin/Product/ProductSeo.php" source-language="en-US" target-language="en" datatype="plaintext">
     <body>
       <trans-unit id="8533d8d42f2c7ef3078797984ea14675">
-        <source>This description will appear in search engines. You need a single sentence, shorter than 160 characters (including spaces)</source>
-        <target>This description will appear in search engines. You need a single sentence, shorter than 160 characters (including spaces)</target>
+        <source>This description will appear in search engines. For optimal referencing you need a single sentence, shorter than 160 characters (including spaces)</source>
```

Comment:
```suggestion
        <source>This description will appear in search engines. For better SEO, write a sentence of 160 characters maximum (spaces included).</source>
```

Reply:
@Julievrz 

> >This description will appear in search engines

will -> may

what do you think? see our conversation about how search engines use meta tags